rmf_site_format/src/mobile_robot.rs

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+use crate::*;
+#[cfg(feature = "bevy")]
+use bevy::prelude::{Bundle, Component};
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[cfg_attr(feature = "bevy", derive(Bundle))]
+pub struct MobileRobot {
+    pub model_name: NameInSite,
+    pub source: AssetSource,
+    #[serde(default, skip_serializing_if = "is_default")]
+    pub scale: Scale,
+    pub kinematics: MobileRobotKinematics,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[cfg_attr(feature = "bevy", derive(Component))]
+pub enum MobileRobotKinematics {
+    DifferentialDrive(DifferentialDrive),
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct DifferentialDrive {
+    pub translational_speed: f32,
+    pub rotational_speed: f32,
```

Comment:
Would it be worth it to add linear / angular accelerations as well? So we can wrap all the data that would be in the fleet adapter `limits` structures

Reply:
I've come to the conclusion that for the next generation of RMF we don't actually want to use the acceleration data. The acceleration numbers provided by users are usually made up, and don't seem to have a meaningful effect on the overall system anyway. Reducing the parameters to just the nominal first-order speed is a strategic choice.

If we need to export backwards-compatible config files, e.g. to be consumed by the current `rmf_fleet_adapter` implementation, we can just automatically fill in 1.0 for linear and 6.0 for angular accelerations.